k8s/api-svc.yaml.template.sh

```diff
@@ -0,0 +1,22 @@
+export APP=mdn-demo-$(git rev-parse --abbrev-ref HEAD)
+eval "cat <<EOF
+apiVersion: v1
+kind: Service
+metadata:
+  labels:
+    app: ${APP}
+  name: api
+  namespace: ${APP}
+spec:
+  ports:
+  - name: http
+    port: 80
+    protocol: TCP
+    targetPort: 8000
+  selector:
+    app: ${APP}
+    heritage: deis
```

Comment:
I don't think it's necessary, but perhaps safer in that we're more specific in targeting the "app: api" container created by Deis via the Procfile? Very thin reason though given that we don't have any pods labelled with "app: api" other than the one created via the Deis Procfile! :)

I'll leave it in for now.